Python/Batch/common/helpers.py

```diff
@@ -265,6 +265,29 @@ def wait_for_all_nodes_state(batch_client, pool, node_state):
         time.sleep(10)
 
 
+def create_container_and_create_sas(
```

Comment:
Why do you need this helper method?  It seems to duplicate the existing `create_sas_token` at the very least it should use that method instead -- but I'd prefer to see it just removed and in your code you can do:

```
block_blob_client.create_container(
    container_name,
    fail_on_exist=False)
helpers.create_sas_token(block_blob_client, container_name, blob_name, permission, expiry)
```